objects.go

```diff
@@ -128,6 +141,39 @@ func (chain *Chain) writeOperation(verb verb, ctx *nftContext, writer io.Writer)
 	fmt.Fprintf(writer, "\n")
 }
 
+func parseChain(words []string) (*Chain, error) {
+	if len(words) < 5 {
+		return nil, fmt.Errorf("chain command should have at least 5 words")
+	}
+	chainObj := &Chain{
+		Name: words[4],
+	}
+	if len(words) > 6 {
+		chainStr, comment := splitComment(strings.Join(words[6:len(words)-1], " "))
+		if comment != nil {
+			chainObj.Comment = comment
+		}
+		if len(chainStr) > 0 {
+			chainWords := strings.Split(chainStr, " ")
```

Comment:
This is awkward... you start with a string in the caller, split it into words, re-join the words to call `splitComment`, then re-split the return value...

maybe `splitComment` should return a `[]string` rather than a `string`... (in which case the simplest implementation probably wouldn't be regexp; you'd just scan for the word "comment", make sure the next word starts with `"`, and scan for a word that ends with `"`.)

Alternatively, you could do everything with regexps on flat strings rather than ever splitting it into words. We probably wouldn't want to do that for "real" data, but since this is just for unit tests it's fine.